src/client/mod.rs

```diff
@@ -108,6 +108,16 @@ pub struct Client {
     inner: Arc<ClientInner>,
 }
 
+#[allow(dead_code, unreachable_code, clippy::diverging_sub_expression)]
```

Comment:
Accidentally making `Client` non-`Send` or -`Sync` causes confusing errors in random other parts of the code where those constraints are implicitly used; this little hack (which appears to be the standard way to assert auto traits) means you get a more clear error.

Reply:
nice 😎